integrators/symplectic_partitioned_runge_kutta_integrator.hpp

```diff
@@ -0,0 +1,343 @@
+﻿
+#pragma once
+
+#include <type_traits>
+
+#include "numerics/fixed_arrays.hpp"
+#include "symplectic_runge_kutta_nyström_integrator.hpp"
+
+namespace principia {
+
+using numerics::FixedVector;
+
+namespace integrators {
+
+// A symplectic partitioned Runge-Kutta integrator.  Does not subclass
+// |Integrator| yet; used to generate (less general)
```

Comment:
Would be good to have a bug to subclass `Integrator`, unless you plan to do it really soon.